subseas_flex_fcst/layout.py

```diff
@@ -29,11 +29,18 @@
 def app_layout():
 
     # Initialization
+    if CONFIG["leads"] != "none":
```

Comment:
If you make the change I suggested in the config file, this will change to `is not None`.